### ISSUE TITLE
Extension no longer available?

### ISSUE THREAD

**Issue by marcinkunert** [nealwu/KillNewsFeed#7]
The link http://killnewsfeed.com/ points to 404 https://chrome.google.com/webstore/detail/kill-news-feed/hjobfcedfgohjkaieocljfcppjbkglfd

Chrome extension management says it violates the Chrome Web Store rules.

**Comment by nealwu** [comment]
Yes, unfortunately Facebook took it down for logo infringement 😔
